plugins/org.jboss.tools.openshift.common.ui/src/org/jboss/tools/openshift/internal/common/ui/explorer/OpenShiftExplorerView.java

```diff
@@ -123,7 +123,8 @@ private Control createExplanationPane(Control connectionsPane, PageBook pageBook
 	private SelectionAdapter onExplanationClicked(final Control connectionsPane, final Control explanationPane) {
 		return new SelectionAdapter() {
 			public void widgetSelected(SelectionEvent e) {
-				ConnectionWizard wizard = new ConnectionWizard();
+				ConnectionWizard wizard = new ConnectionWizard();				
+				wizard.setWindowTitle("New OpenShift Connection");
```

Comment:
@adietish  I'd thought about putting it into the constructor, but then I spotted this wizard is used also for modification (Edit) of an existing connection, where is different title, so I put it there to let it be in responsibilities of the one who is using this wizard to put correct title for it. But if you want, I could put it to constructor to let it be default and who wanna reuse the wizard would have to call set title method.